packages/server/src/api-docs/open-api.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
instead of adding comments to the routes/users.ts file, we can add an open-api.json file to the project which helps in easily creating the API paths. vscode extensions (https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi)  play nicely with the JSON files and provide IntelliSense. 

Adding comments on top of the routes is not so easy as vscode is not providing any help and we have to add each and every line in the comment.